azerothcore/azerothcore-wotlk
# fix(Core/mail): letter paper

## Changes Proposed:
![image](https://user-images.githubusercontent.com/18535853/218616325-e5ada35c-971c-48a7-8cce-854dfb8b566c.png)
to
![image](https://user-images.githubusercontent.com/18535853/218616234-66d42d46-932b-466b-a4c1-79199b04cb6e.png)
-  

## Issues Addressed:

- Closes 

## SOURCE:


## Tests Performed:

- 
- 


## How to Test the Changes:


1. Before PR, open the mailbox and save the mail
2. After PR, open the mailbox and save the message
3. Check the change of stationery in the backpack

## Known Issues and TODO List:




## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
